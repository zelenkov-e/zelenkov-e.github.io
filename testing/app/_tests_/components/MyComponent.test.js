import React from 'react';
import { shallow } from 'enzyme';
import { MyComponent } from './MyComponent';
// import { shallowToJson  } from 'enzyme-to-json';



describe("MyComponent", () => {
  it(" render my component", () => {
    const wrapper = shallow(<MyComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});