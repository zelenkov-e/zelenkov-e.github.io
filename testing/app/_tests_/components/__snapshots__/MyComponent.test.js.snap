// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent  render my component 1`] = `
<div>
  <input
    type="text"
  />
</div>
`;
